src/main/java/com/artipie/asto/fs/FileStorage.java

```diff
@@ -118,11 +120,26 @@ public FileStorage(final Path path) {
             .toCompletableFuture();
     }
 
+    @Override
+    public CompletableFuture<Void> move(final Key source, final Key destination) {
+        return Completable.fromCallable(
+            () -> {
+                final Path from = this.path(source);
+                final Path dest = this.path(destination);
+                dest.getParent().toFile().mkdirs();
+                Files.move(from, dest, StandardCopyOption.REPLACE_EXISTING);
+                return null;
+            })
+            .to(CompletableInterop.await())
+            .<Void>thenApply(o -> null)
```

Comment:
@olegmoz What is ``o`` here? Can you give a proper variable name? In this context I really have no idea what it is.

Reply:
@amihaiemil renamed it to 'file', but to be honest you should not care what is in this variable at this point. Only reason for this code is to convert <RxFile> to <Void>, nothing else